cmake/modules/FindDEAL_II_OPENCASCADE.cmake

```diff
@@ -66,11 +66,19 @@ if(EXISTS ${OPENCASCADE_INCLUDE_DIR}/Standard_Version.hxx)
 endif()
 
 # These seem to be pretty much the only required ones.
-set(_opencascade_libraries
-  TKBO TKBool TKBRep TKernel TKFeat TKFillet TKG2d TKG3d TKGeomAlgo
-  TKGeomBase TKHLR TKIGES TKMath TKMesh TKOffset TKPrim TKShHealing TKSTEP
-  TKSTEPAttr TKSTEPBase TKSTEP209 TKSTL TKTopAlgo TKXSBase
-  )
+if(OPENCASCADE_VERSION AND OPENCASCADE_VERSION VERSION_GREATER_EQUAL "7.8.0")
+  set(_opencascade_libraries
+    TKBO TKBool TKBRep TKernel TKFeat TKFillet TKG2d TKG3d TKGeomAlgo
+    TKGeomBase TKHLR TKDEIGES TKMath TKMesh TKOffset TKPrim TKShHealing TKDESTEP
+    TKDESTL TKTopAlgo TKXSBase
+    )
+    else()
+  set(_opencascade_libraries
+    TKBO TKBool TKBRep TKernel TKFeat TKFillet TKG2d TKG3d TKGeomAlgo
+    TKGeomBase TKHLR TKIGES TKMath TKMesh TKOffset TKPrim TKShHealing TKSTEP
+    TKSTEPAttr TKSTEPBase TKSTEP209 TKSTL TKTopAlgo TKXSBase
+    )
+endif()
```

Comment:
Would you mind to refactor this into the following logic?
```
set(_opencascade_libraries <common components>)
if(OPENCASCADE_VERSION AND OPENCASCADE_VERSION VERSION_GREATER_EQUAL "7.8.0")
  list(APPEND _opencascade_libraries <new components here>
else()
  list(APPEND _opencascade_libraries <old components here>
endif()
```
This would make it easier to understand what is going on.